imagersite/imager_images/views.py

```diff
@@ -25,9 +25,9 @@ class AlbumFormView(FormView):
 
     def get_form(self, form_class=AlbumForm):
         try:
-            album = Album.objects.get(user=self.request.user)
+            album = Album.objects.get(user=self.request.user, pk=self.kwargs['pk'])
```

Comment:
I believe the `CreateView` version of the django generic CBVs will take care of pk for you.  It's a bit like a mix of the FormView and the DetailView.